### ISSUE TITLE
Is there any plans for pause/restart for Android?

### ISSUE THREAD

**Issue by StephanCassel** [dlutton/flutter_tts#113]
Hi 
Excellent plugin. Very useful indeed.
Is there any plans for pause/restart for Android?

**Comment by dlutton** [comment]
@StephanCassel the [Android TTS SDK](https://developer.android.com/reference/android/speech/tts/TextToSpeech) doesn't have a pause/restart function at this time.

**Comment by StephanCassel** [comment]
@dlutton Thanks for the replay.

I solved the pause/play by splitting up a large text into substrings (substring == sentence) and a "sentence cursor". flutterTts.speak (first sentence). setCompletionHandler () restarts next sentence and so forth. When I stop the speak and restart the **current** sentence at the sentence cursor.
It turns out that it was a good solution for the user. The user was able to recap the content. Just start in a middle of sentence is not so user friendly.

**Comment by dlutton** [comment]
Depending on how you're using it, you can also use the android synthesize to file feature and use the media play/pause/stop as it'll get saved as a .wav file.  Once Android supports play/pause in its TTS SDK, I'll implement it in this plugin.

**Comment by StephanCassel** [comment]
Yes, that could be a good solution. I will have that in mind.
For this projects need the "rewind" to start of a sentence is in fact very good. 
It works like a charm. 
Thanks for the tips!

**Comment by Pherne** [comment]
I'm sorry if I'm very late to this, but how would I use the file I created? I have a newspaper app where I want the app to read an article on button tap and I also want it to be able to pause and resume.

This is what I tried

`              
var file = 'articleText.wav';
await flutterTts.synthesizeToFile(text, file);
print('$dir$file');

var duration = await player.setFilePath('$dir$file');
await player.play();`

But I get this error:

`Caused by: com.google.android.exoplayer2.source.UnrecognizedInputFormatException: None of the available extractors (WavExtractor, FlvExtractor, FlacExtractor, FragmentedMp4Extractor, Mp4Extractor, AmrExtractor, PsExtractor, OggExtractor, TsExtractor, MatroskaExtractor, AdtsExtractor, Ac3Extractor, Ac4Extractor, Mp3Extractor, JpegExtractor) could read the stream.`

I'm aware that my question includes another plugin. It's just that the audio plugin can play other sound files just fine and I have the same issue with another audio plugin. So I'm assuming it's related to the file I created. However when I adb pull the file and send it to my phone it can play the audio.
